lib/pluginmanager/bundler/logstash_injector.rb

```diff
@@ -67,7 +67,7 @@ def inject(gemfile_path, lockfile_path, dependencies)
       gemfile = LogStash::Gemfile.new(File.new(gemfile_path, "r+")).load
 
       begin
-        @new_deps.each do |dependency|
```

Comment:
this is simply due to a change in the name of the variable in recent bundlers:
- 1.17.1 uses `@deps` https://github.com/bundler/bundler/blob/v1.17.1/lib/bundler/injector.rb#L18
- 1.9.10 used `@new_deps` https://github.com/bundler/bundler/blob/v1.9.10/lib/bundler/injector.rb#L9